### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by bradymiller**
remove a lone semicolon
sql fix for limit
added document name to labels for patient photographs (to be consistent with the insurance cards which fixed to show the document name label instead of the uuid) in patient summary

### CODE REVIEW

**Comment by stephenwaite** [comment]
jumping

File: interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/CarecoordinationTable.php [hunk 1]
```diff
@@ -766,7 +766,6 @@ public function insert_patient($audit_master_id, $document_id)
                 $e++;
             } elseif ($table == 'functional_cognitive_status') {
                 $arr_functional_cognitive_status['functional_cognitive_status'][$i]['cognitive'] = $newdata['functional_cognitive_status']['cognitive'];
-                ;
```
Comment (sjpadgett) [inline interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/CarecoordinationTable.php:769]: this is already done in my PR!

Comment (bradymiller) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/CarecoordinationTable.php:769]: lol, i will drop this fix :)

**Comment by bradymiller** [comment]
testing well and bringing in

**Pull request merged**